syspage.c

```diff
@@ -391,23 +391,30 @@ void syspage_showMaps(void)
 		lib_printf("%d%-3s %-8s 0x%08x%4s 0x%08x%4s 0x%08x%4s 0x%08x%4s %s\n", pmap->map.id, "", pmap->map.name,
 			pmap->map.start, "", pmap->map.end, "", pmap->top, "", pmap->map.end - pmap->top, "", attr);
 	}
+
+	hal_memInfo();
 }
 
 
 void syspage_showApps(void)
 {
 	int i;
 	syspage_program_t *prog;
+	const char *imap, *dmap;
+
 	if (syspage_common.progsCnt == 0) {
 		lib_printf("\nApps number: 0");
 		return;
 	}
 
-	lib_printf(CONSOLE_BOLD "\n%-16s %-14s %-14s %-14s %-14s\n", "NAME", "START", "END", "IMAP ID", "DMAP ID");
+	lib_printf(CONSOLE_BOLD "\n%-16s %-14s %-14s %-9s %-9s\n", "NAME", "START", "END", "IMAP ID", "DMAP ID");
 	lib_printf(CONSOLE_NORMAL);
 	for (i = 0; i < syspage_common.progsCnt; ++i) {
 		prog = &syspage_common.progs[i];
-		lib_printf("%-16s 0x%08x%4s 0x%08x%4s %d%13s %d\n", prog->name, prog->start, "", prog->end, "", prog->imap, "", prog->dmap);
+		imap = syspage_getMapName(prog->imap);
+		dmap = syspage_getMapName(prog->dmap);
+		lib_printf("%-16s 0x%08x%4s 0x%08x%4s %-9s %-9s\n",
+			prog->name, prog->start, "", prog->end, "", imap ? imap : "<invalid>", dmap ? dmap : "<invalid>");
 	}
```

Comment:
Formatting and all stuff related to displaying should be defined by Hubert.

Reply:
Ack.

Reply:
Presenting maps as string instead of number has been discussed with me. We assumed that it is better approach. Using `app` command user already knows the name of map, it is more intuitive. 

However, there have been already checked that apps use valid maps, so that there don't have to be used `syspage_getMapName` function which search through map's list. Instead of that, the map's name can be got based on `id`.

Reply:
Changed to what @HBuczynski suggested, get map name by `id`, however `syspage_getMapName()` is used in `mpu` command to get linked by `id` map name.